main.tex

```diff
@@ -171,11 +182,9 @@
 
 \begin{enumerate}[label=(\alph*)]
 
-\item  notify the Manager or the Owners Corporation of the danger or threat;
-
-\item  obey the Manager or the Owners Corporation instructions, including evacuating the Lot or the Common Property; and
+\item Notify the relevant emergency service
 
-\item  obey the instructions of the policy, fire brigade or relevant Authority.
```

Comment:
Governments already have rules about following orders of emergency services. We do not need Owners Corporations rules for this.